week6.3-ex4.py

```diff
@@ -0,0 +1,27 @@
+from typing import List
+FIRST_NAMES = ['avi', 'moshe', 'yaakov']
+LAST_NAMES = ['cohen', 'levi', 'mizrahi']
+
+
+def full_names(first_names: List[str], last_names: List[str], min_length: int = 0) -> List:
+    """
+    The function gets a list of first and last names and creates full names,
+       and becomes a capital letter every first letter in the name.
+    :param first_names: List of first names.
+    :param last_names: List of last names
+    :param min_length: Minimum length for full name, the default value is 0, meaning any length is obtained.
+    :return: The list of full names.
+    """
+    return [first_name[0].upper() + first_name[1:] + ' ' + last_name[0].upper() + last_name[1:]
```

Comment:
I would exmport the logic in this list comprehetion to a lambda , or to a separate function (without documentation). In my taste this one liner is a little bit too long, but thats a matter of opinion